test/integration/test_i_aws_encrytion_sdk_client.py

```diff
@@ -53,19 +53,26 @@ def test_encrypt_verify_user_agent_kms_master_key(caplog):
     caplog.set_level(level=logging.DEBUG)
     mk = KMSMasterKey(key_id=get_cmk_arn())
 
-    mk.generate_data_key(algorithm=Algorithm.AES_256_GCM_IV12_TAG16_HKDF_SHA384_ECDSA_P384, encryption_context={})
+    mk.generate_data_key(algorithm=AlgorithmSuite.AES_256_GCM_IV12_TAG16_HKDF_SHA384_ECDSA_P384, encryption_context={})
 
     assert USER_AGENT_SUFFIX in caplog.text
 
 
 def test_remove_bad_client():
     test = KMSMasterKeyProvider()
     test.add_regional_client("us-fakey-12")
-
     with pytest.raises(BotoCoreError):
         test._regional_clients["us-fakey-12"].list_keys()
 
-    assert not test._regional_clients
+    # I believe that because KMSMasterKeyProvider() sets a default regional client
```

Comment:
Let's figure this out for sure and make the comment more declarative. "I believe" implies to me that we're not quite sure.

Reply:
Changed the comment to be more concise and declarative.